tutorials/SpectroscopicDataReductionBasics/2-WavelengthCalibration.ipynb

```diff
@@ -0,0 +1,1725 @@
+{
```

Comment:
<p>Spectroscopic Data Reduction 2. No "Part".  and refer to actual title of other tutorial and somehow link?  I guess we have to publish the first one to put in the links. The notebook assumes you've completed Spectroscopic Data Reduction 1: Finding the Trace or have a trace model handy.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
You don't like "part"?

For linking, yeah, that's an infrastructure question - I'd like to request that get punted back to the infrastructure team

Reply:
I added a link but it may need to be fixed.